release.json

```diff
@@ -1,12 +1,12 @@
 {
   "nio/S3Download": {
     "language": "Python",
-    "version": "0.0.1",
-    "url": "git://github.com/nio-blocks/S3.git"
+    "url": "git://github.com/cowleyk/amazonS3.git",
```

Comment:
This now points to your fork of S3, also leave the version tweaking for ol' man Jenkins